File: null-log/src/arguments/version_message.f90
module version_message
  implicit none

  !> Print Version Message
  function print_version() result(value)
    integer :: value
    character(len=5) :: version_number

    version_number = "1.0.0"
    
    print "(A)", "\x1b[32;1;3;4mNull Log Tool\x1b[0m"
    print "(A)", ""
    print "(A)", "\x1b[32;1;3mVersion Number:\x1b[0m\t", version_number

    value = 0
  end function print_version

end module version_message

File: gh-data/src/arguments/version_message.c
#include "./version_message.h"

#include <stdint.h>
#include <stdio.h>

// Print Version Message
uint8_t print_version(void) {
  char version_number[6] = u8"1.0.0";

  printf("\x1b[32;1;3;4mGithub Data Tool\x1b[0m\n\n");
  printf("\x1b[32;1;3mVersion Number:%s\x1b[0m\n", version_number);

  return 0;
}
